server/mvc/models/UtilityModel.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+/**
+ * A model containing utility tools, e.g. 'config' data, loggers, etc.
+ */
+var inherit = require("inherit");
+var DefaultModel = require("./DefaultModel");
+
+var config;
+var log;
+
+var UtilityModel = inherit(DefaultModel, {
+  getConfig: function(){
+    return UtilityModel.getConfig();
+  },
+  getLog: function(){
+    return UtilityModel.getLog();
+  },
+}, {
+  getConfig: function(){
+    return config;
```

Comment:
@Sikian @theevenprime2 
This, for instance, is important. If one of us developers accidentally adds a piece of code that dumps an instance of the `UtilityModel` - which of course is an Object - and it is somehow visible to our users, all the users will see is that `getConfig` is a function. They cannot see what is inside the `config` data. This adds another layer of security.